src/Playroom/Toolbar/Toolbar.css.ts

```diff
@@ -92,24 +92,52 @@ export const panel = style([
     position: 'relative',
     display: 'flex',
     overflow: 'auto',
-    transition: 'slow',
     pointerEvents: 'auto',
   }),
   {
     width: toolbarOpenSize,
     backgroundColor: colorPaletteVars.background.surface,
     borderLeft: `${toolbarBorderThickness} solid ${colorPaletteVars.border.standard}`,
-    selectors: {
-      [`${root}:not(${isOpen}) &`]: {
-        transform: `translateX(${calc(`${toolbarOpenSize}px`).multiply(0.3)})`,
-        opacity: 0,
-        pointerEvents: 'none',
-      },
-    },
   },
 ]);
 
 export const preference = sprinkles({
   position: 'absolute',
   inset: 0,
 });
+
+export const transitionStyles = {
+  enter: style({
+    opacity: 0,
+    transform: `translateX(${calc(`${toolbarOpenSize}px`).multiply(0.3)})`,
```

Comment:
Worth bringing this out into a constant so we can name it and avoid the duplication.

Reply:
I've refactored slightly, but I think it might be better to keep the full styles here so that it's easier to compare the differences in each transition stage, and looks like the more common way this library is used. Thoughts?

Reply:
Oh, I was thinking extracting the entire string into a constant, but I think using `translateX(30%)` is better than referencing `toolbarOpenSize`. Nice.